static/src/javascripts/projects/common/modules/experiments/tests/membership-message.js

```diff
@@ -37,8 +37,8 @@ define([
              */
             var alreadyVisited = storage.local.get('alreadyVisited') || 0;
             return !detect.adblockInUse &&
-                config.page.edition.toLowerCase() === 'uk' &&
-                config.page.contentType.toLowerCase() === 'article' &&
+                config.page.edition === 'UK' &&
+                config.page.contentType === 'Article' &&
```

Comment:
true, although it does concern me a tiny bit that you have to lower case things to compare them, surely we can have a defined case for things?  not a major issue but it adds complexity to the code having all these extra steps